src/components/editTestCase/groupCoverage/GroupCoverage.tsx

```diff
@@ -85,41 +104,49 @@ const GroupCoverage = ({ groupPopulations, calculationResults }: Props) => {
       });
   };
 
-  const getPopulationResults = (groupId: string): Array<PopulationResult> => {
-    const groupCalculations = calculationResults?.find(
-      (result) => result.groupId === groupId
-    );
-    if (groupCalculations) {
-      const relevantPopulations = groupCalculations.populationRelevance;
-      const statementResults = groupCalculations.statementResults;
-      return relevantPopulations.map((population) => {
-        const populationResult = statementResults.find(
-          (statementResult) =>
-            statementResult.statementName === population.criteriaExpression
-        );
-        return {
-          ...populationResult,
-          populationName: FHIR_POPULATION_CODES[population.populationType],
-        };
-      });
+  const getPopulationResults = (groupId: string) => {
+    if (mappedCalculationResults) {
+      const selectedGroupCalculationResults = mappedCalculationResults[groupId];
+      if (selectedGroupCalculationResults) {
+        const relevant1Populations =
```

Comment:
What does the `1` in `relevant1Populations` represent?

Reply:
I was testing multiple way in which I was trying to get the required output so I named the relevantPopulations, relevant1Populations and after finalizing the method I forgot to remove the number. I'll remove it